### DISCUSSION TITLE
Jackbox -> Discord

### DISCUSSION THREAD

**Discussion by hujiba** [ExistentialAudio/BlackHole#90] [General]
I use Discord to stream a game to friends. On Discord you can choose the input device, but on jackbox you can’t choose an output. I want to be able to have the people in discord to be able to hear the direct game sound, but also my voice. I am on Catalina. How can I do this? If i need to use a DAW, that’s fine, but I’m not sure exactly how to use BlackHole within the DAW. Thanks!

**Comment by devinroth** [comment]
I use Logic Pro X but any daw should work. Setup an aggregate device and route audio to both your speakers and to BlackHole. Input audio from BlackHole to Discord. That should do the trick.

**Comment by hujiba** [comment]
Thank you, but I’m still not sure where the daw comes in. How I route the audio using a DAW? I have logic pro, so I can use that.

**Comment by devinroth** [comment]
Set the input/output devices in logic to your aggregate. Create an audio track and set the input to to your microphone and output to BlackHole. Make sure you record enable the track and click input monitoring. This will send your mic to BlackHole. Then use create a multi-output and set it to your system output. Now you should have your mic and system audio going to Discord. Beware, Discord only lets mono audio in.

**Comment by hujiba** [comment]
Thank you!

**Comment by itsclutch** [comment]
So I have the same problem but I am a bit bad with technology. Are there some easier steps I can follow? Got a bit lost on the first step.

**Comment by hujiba** [comment]
> So I have the same problem but I am a bit bad with technology. Are there some easier steps I can follow? Got a bit lost on the first step.
> 
> 

Honestly I still don't really know how to do it. I actually just got a windows pc after I posted that so that solved the problem for me. Nevertheless, I couldn't figure out how to set the logic pro inputs and outputs different on a single track. I could only do it through preferences (or set track input as Input 1, which didn't do anything except generate a lot of feedback). I don't need this anymore, but it would be nice for other people to have clarification ore a more detailed walk through. Or even a way to use audacity, since it's a simpler program.

**Comment by itsclutch** [comment]
Never mind, I got it to work! I'm using Garageband as my DAW, but I basically went to preferences -> Audio/MIDI and changed the output device to Blackhole and the input device to Built-in Microphone. On Discord I put the input as Blackhole.

**Comment by hujiba** [comment]
> Never mind, I got it to work! I'm using Garageband as my DAW, but I basically went to preferences -> Audio/MIDI and changed the output device to Blackhole and the input device to Built-in Microphone. On Discord I put the input as Blackhole. 

👍
